site/snippets/session-keys/add-session-key.ts

```diff
@@ -1,8 +1,10 @@
 import { SessionKeyPermissionsBuilder } from "@alchemy/aa-accounts";
 import { client } from "./base-client.js";
+import { keccak256 } from "viem";
 
 const result = await client.addSessionKey({
   key: "0xSessionKeyAddress",
-  tag: "0xkeytag",
+  // tag is an identifier for the emitted SessionKeyAdded event
+  tag: keccak256(new TextEncoder().encode("your-app-tag")),
```

Comment:
it's tag for a single session key
```suggestion
  tag: keccak256(new TextEncoder().encode("session-key-tag")),
```